src/darwin/Framework/CHIP/MTRDeviceController_Concrete.mm

```diff
@@ -358,6 +361,15 @@ - (void)_controllerSuspended
     MTRDeviceControllerFactory * factory = _factory;
     dispatch_async(_chipWorkQueue, ^{
         factory.operationalBrowser->ControllerDeactivated();
+
+        if (self.shouldAdvertiseOperational) {
+            auto * fabricTable = factory.fabricTable;
+            if (fabricTable) {
+                // We don't care about errors here. If our fabric is gone, nothing to do.
+                fabricTable->SetShouldAdvertiseIdentity(self->_storedFabricIndex, chip::FabricTable::AdvertiseIdentity::No);
+                [factory resetOperationalAdvertising];
```

Comment:
Are there any risks if `app::DnssdServer::Instance().StartServer()` is called consecutively multiple times? Thinking about edge case where suspend/resume is called in rapid succession.

Reply:
It might end up sending out some extra packets, in theory, but apart from that, no.